share/src/main/java/io/zensoft/share/service/FacebookPublisherService.java

```diff
@@ -0,0 +1,121 @@
+package io.zensoft.share.service;
+
+import io.zensoft.share.model.Vacancy;
+import io.zensoft.share.model.VacancyResponse;
+import io.zensoft.share.service.model.VacancyResponseModelService;
+import io.zensoft.share.service.model.VacancyModelService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.social.facebook.api.Facebook;
+import org.springframework.social.facebook.api.PagePostData;
+import org.springframework.social.facebook.api.impl.FacebookTemplate;
+import org.springframework.stereotype.Service;
+import org.springframework.web.client.RestTemplate;
+
+import java.io.FileReader;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Properties;
+
+public class FacebookPublisherService {
+    private final Properties properties;
+
+    private final String appAccessToken;
+    private final String userAccessToken;
+    private final String pageAccessToken;
+
+    private final Facebook facebookApp;
+    private final Facebook facebookUser;
+    private final Facebook facebookPage;
+
+    public FacebookPublisherService() throws Exception {
+        properties = new Properties();
+        properties.load(new FileReader("src/main/java/io/zensoft/share/config/Facebook/facebookConfig"));
+
+        appAccessToken = properties.getProperty("appId") + "|" + properties.getProperty("appSecret");
+        facebookApp= new FacebookTemplate(appAccessToken, properties.getProperty("appNamespace"), properties.getProperty("appId"));
+
+        userAccessToken = getTestUserAccessToken();
+        facebookUser = new FacebookTemplate(userAccessToken, properties.getProperty("appNamespace"), properties.getProperty("appId"));
+
+        pageAccessToken = getPageAccessToken();
+        facebookPage = new FacebookTemplate(pageAccessToken, properties.getProperty("appNamespace"), properties.getProperty("appId"));
+    }
+
+
+    public static void main(String[] args) throws Exception {
```

Comment:
i think we have already main method)

Reply:
Yes, I add it to work with my class. I'll remove it now. Thank you for your comment!